src/Microsoft.IdentityModel.Tokens/CryptoProviderCacheOptions.cs

```diff
@@ -0,0 +1,53 @@
+﻿//------------------------------------------------------------------------------------------------
+// <copyright file="TokenValidator.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//-------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.IdentityModel.Logging;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// Specifies the CryptoProviderCacheOptions which can be used to configure the internal cryptoprovider cache.
+    /// For the net461, net472, and netstandard2.0 targets we are using the Microsoft.Extensions.Caching.Memory.MemoryCache class:
+    /// https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.caching.memory.memorycache
+    /// For the net45 target we are using our own simple caching implementation which clears the entire cache
+    /// once the cache is at max capacity. We recommend upgrading to net461+ for a more comprehensive caching experience.
+    /// Any property on these CryptoProviderCacheOptions that corresponds directly to a property 
+    /// used by Microsoft.Extensions.Caching.Memory.MemoryCache has the same name.
+    /// </summary>
+    public class CryptoProviderCacheOptions
+    {
+        private int _sizeLimit = DefaultSizeLimit;
+
+        /// <summary>
+        /// Default value for <see cref="SizeLimit"/>.
+        /// </summary>
+        public static readonly int DefaultSizeLimit = 1000;
+
+        /// <summary>
+        /// Gets or sets the size of the cache (in number of items). 
+        /// On the net45 target, the entire cache will be cleared once the cache reaches max capacity.
```

Comment:
comment is out of date

Reply:
Fixed here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1605/commits/af3243229ca3f4885398299f0d733066d3b9be46.